db/seeds.rb

```diff
@@ -12,175 +12,171 @@
 # journey3 = Journey.create!(start_date:Date.new(2022,12,02), end_date:Date.new(2022,12,10), tag: "current journey", comment: "current holiday", name: "journey3", user: User.first)
 # journey4 = Journey.create!(start_date:Date.new(2023,12,14), end_date:Date.new(2023,12,21), tag: "future journey", comment: "future holiday", name: "journey4", user: User.first)
 
-#paris = City.create(name: "Paris",
+# paris = City.create(name: "Paris",
 #            description: "Oo la la",
-#            latitude: 48.864716,
-#            longitude: 2.349014,
+#            latitude:  48.864716,
+#            longitude:  2.349014,
 #            region: "ile-de-france",
 #            country: "France",
 #            continent: "Europe",
 #            tags: "Romantic, Historical")
 
-paris = City.create(name: "Paris",
```

Comment:
Below i have changed syntax for each city.next_cities = %w[cities go here]

This is such that first and last letter aren't removed as before.